src/Text/Pandoc/Writers.hs

```diff
@@ -25,7 +25,6 @@ module Text.Pandoc.Writers
     , writeBibLaTeX
     , writeCommonMark
     , writeConTeXt
-    , writeCustom
```

Comment:
Since any API change might break something, I'd lean towards being conservative and leaving this here.  It's not doing any harm, is it? It's true that it's not consistent with Text.Pandoc.Readers, but a less intrusive change to restore consistency would be to export `readCustom` from that module.

Reply:
I probably should have put this in the commit message: the *reason* for removing it is to avoid a circular dependency (Writers -> Writers.Custom -> Lua -> Writers). Reader/Writer consistency is just my justification for why I believe this change is ok.

Reply:
oh, I see. Well, that's a stronger reason!  (And that's probably the reason readCustom is not in T.P.Readers)